### PR TITLE
VSSDK update

### PR DESCRIPTION
**Pull request by ivanbasov**

### CODE REVIEW

**Comment by genlu** [comment]
build in Integration tests are broken


`##[error]src\Setup\DevDivInsertionFiles\DevDivInsertionFiles.csproj(141,5): error : Unexpected dependencies found -- update ExpectedDependency list: Microsoft.Win32.Registry;System.Security.AccessControl;System.Security.Principal.Windows
`

**Comment by ivanbasov** [comment]
> build in Integration tests are broken
> 
> `##[error]src\Setup\DevDivInsertionFiles\DevDivInsertionFiles.csproj(141,5): error : Unexpected dependencies found -- update ExpectedDependency list: Microsoft.Win32.Registry;System.Security.AccessControl;System.Security.Principal.Windows `

Thank you, @genlu ! Fixed.

File: src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj [hunk 1]
```diff
@@ -66,6 +66,10 @@
       TODO: add OptimizeAssemblies="lib/netstandard2.0/System.Threading.Tasks.Extensions.dll"
     -->
     <ExpectedDependency Include="System.Threading.Tasks.Extensions" />
+
+    <ExpectedDependency Include="Microsoft.Win32.Registry"/>
```
Comment (genlu) [inline src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj]: We probably needs somebody to confirm this change is OK.

Comment (ivanbasov) [reply src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj]: @tmat could you please take a look if it is a valid update of `ExpectedDependency`?

File: src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj [hunk 2]
```diff
@@ -66,6 +66,10 @@
...
+    <ExpectedDependency Include="System.Security.AccessControl"/>
+    <ExpectedDependency Include="System.Security.Principal.Windows"/>
```
Comment (tmat) [inline src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj]: This is incorrect. We do not own insertion of these assemblies.

**Comment by tmat** [comment]
>       <_Dependency Remove="VSLangProj80"/>

List the assemblies here instead.

---
Refers to: src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj:124 in 1bbcd4f. [](commit_id = 1bbcd4fa192657ef43b4522fce2033e592116ac0, deletion_comment = False)

File: src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj (hunk 2 above)
Comment (ivanbasov) [reply src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj]: Thank you, @genlu and @tmat ! Updated with `<_Dependency Remove=.../>`

**Comment by ivanbasov** [comment]
Feedback addressed. @tmat and @jasonmalinowski please review

**Comment by ivanbasov** [comment]
> Looks good to me, but please make sure @tmat signs off on the NuGet packaging configuration stuff.

Thank you, @jasonmalinowski ! Agree for @tmat sign off. @tmat , could you please take a look?

**Comment by ivanbasov** [comment]
@jinujoseph FYI this contains only SDK updates and no code or tests changes. It has been merged into master-vs-deps. Now I should be unblocked with Go To Base UI.

**Pull request merged**